frontend/lib/wallets/MetaMask.ts

```diff
@@ -10,7 +11,8 @@ export default class MetaMask extends AbstractWallet {
     public static icon = MetaMaskLogo;
     public static downloadUrl = 'https://metamask.io/';
 
-    private signer?: ethers.providers.JsonRpcSigner;
+    private static provider?: any;
```

Comment:
First of all, why do you now need to have static variable that holds provider? Can this instead be a static [getter] function?

Can you please avoid using `any` as much as possible? Types are helpful when you use them to reinforce your code. What do you plan to store here?

Reply:
according to

> First of all, why do you now need to have static variable that holds provider? 

and [this](https://github.com/sidestream-tech/unified-auctions-ui/pull/562#discussion_r1045661166) and [this](https://github.com/sidestream-tech/unified-auctions-ui/pull/562#issuecomment-1347918447)

i've merged the static variable and function into static getter function.